packages/build-deb-packages.sh

```diff
@@ -41,56 +41,60 @@ if [ "X$VERSION" == "X" ]; then
 fi
 FULL_VERSION="${VERSION}-${DEBIAN_VERSION}"
 
-tmpdir=`mktemp -d /tmp/gdr.XXXXXX`
-if [ ! -d "$tmpdir" ]; then
+tmpdir=$(mktemp -d /tmp/gdr.XXXXXX)
+if [ ! -d "${tmpdir}" ]; then
     echo "Failed to create a temp directory!" >&2
     exit 1
 fi
 
 echo "Building gdrcopy debian packages version ${FULL_VERSION} ..."
 
-echo "Working in $tmpdir ..."
+echo "Working in ${tmpdir} ..."
 
 ex cd ${TOP_DIR_PATH}
 
-ex mkdir -p $tmpdir/gdrcopy
```

Comment:
can you separate the ${tmpdir} cleanup in a separate patch within this same PR ?
it makes review much easier